featurebyte/schema/target_namespace.py

```diff
@@ -24,11 +24,11 @@ class TargetNamespaceCreate(FeatureByteBaseModel):
 
     id: Optional[PydanticObjectId] = Field(default_factory=ObjectId, alias="_id")
     name: StrictStr
-    dtype: DBVarType
+    dtype: DBVarType = Field(default=DBVarType.UNKNOWN)
```

Comment:
Should we make this `Optional[DBVarType]`?

Reply:
Or `dtype` should be a required parameter during target namespace creation

Reply:
hm i think there's no requirement for the dtype to be known during namespace creation at the moment. will update to optional!

Reply:
If there is no requirement, we should use `Optional`. Thinking whether we should check with Xavier whether to include this.